src/main/extension/extension_helper.cpp

```diff
@@ -116,11 +116,13 @@ static const DefaultExtension internal_extensions[] = {
     {"spatial", "Geospatial extension that adds support for working with spatial data and functions", false},
     {"aws", "Provides features that depend on the AWS SDK", false},
     {"azure", "Adds a filesystem abstraction for Azure blob storage to DuckDB", false},
+    {"encodings", "All unicode encodings to UTF-8", false},
     {"iceberg", "Adds support for Apache Iceberg", false},
     {"vss", "Adds indexing support to accelerate Vector Similarity Search", false},
     {"delta", "Adds support for Delta Lake", false},
     {"fts", "Adds support for Full-Text Search Indexes", false},
     {"ui", "Adds local UI for DuckDB", false},
+    {"encodings", "All unicode encodings to UTF-8", false},
```

Comment:
Why two?
I would assume this is not visible, but maybe it could go.

Reply:
I did not follow the `Why two?` question.

Reply:
I see two added `encodings` lines: https://github.com/duckdb/duckdb/pull/17206/files/2ac7bb50d738c2725bb97c03c982971949008b78#diff-4ac170a8a0bd128a68535829051a901790ba4ae905e2bf2e7e4651b7cee0a60b

Reply:
ahhh, yeah that's a woopsie daisy